clayui.com/package.json

```diff
@@ -5,10 +5,10 @@
 	"license": "MIT",
 	"scripts": {
 		"copy:clay-css": "yarn copy:clay-css-icons && yarn copy:clay-css-js && yarn copy:clay-css-site-images && yarn copy:clay-css-images",
-		"copy:clay-css-icons": "cpx '../node_modules/@clayui/css/lib/images/icons/*.svg' static/images/icons && node svg-icons.js",
-		"copy:clay-css-js": "cpx '../node_modules/@clayui/css/lib/js/*.{js,js.map}' static/js",
-		"copy:clay-css-images": "cpx '../node_modules/@clayui/css/src/images/images/*.{gif}' static/images",
-		"copy:clay-css-site-images": "cpx '../node_modules/@clayui/css/src/content/site-images/*.svg' static/images",
+		"copy:clay-css-icons": "node svg-icons.js",
```

Comment:
Maybe we could also run svg-icons through the plugin.

Reply:
I'll update the plugin to do this.

Reply:
👍